core/themes/bartik/css/style.css

```diff
@@ -574,13 +574,16 @@ ul.tips {
 .js .menu-dropdown a:hover,
 .js .menu-dropdown a:focus,
 .js .menu-dropdown a:active,
-/* Prevents 2+ level menu items with child items from "jumping" when their 
+/* Prevents 2+ level menu items with child items from "jumping" when their
    child items are hovered over. */
 .js .menu-dropdown .has-children a {
   height: 2.4em;
   line-height: 2.4em;
   padding: 0 0.8em;
 }
+.js .menu-dropdown .has-children a {
```

Comment:
Can we please add a comment here, to explain that this is to avoid the arrow icon overlapping menu item text. Would provide hint and prevent accidentally removing this in the future, causing this to regress.

Also, since this is direction-specific, we should add a `padding-left` on RTL?